eu_dgc_v1_schema.json

```diff
@@ -61,7 +82,7 @@
                 },
                 "gen": {
                     "title": "Administrative gender",
-                    "description": "Administrative gender (optional, viz. VS-2021-04-08)",
+                    "description": "Administrative gender (optional, viz. VS-2021-04-14)",
                     "type": "string",
                     "example": "male"
```

Comment:
male/female. Too long. Why not M, F and U?

Reply:
male/female ... based on "Document "Value Sets for the digital green certificate as stated in the Annex ..." https://webgate.ec.europa.eu/fpfis/wikis/x/05PuKg"

Reply:
If the allowed values are limited, please declare it as a enum.

Reply:
The value sets for all properties, where i noted "VS-2021-04-14" in the description, where defined by the Subgroup "Semantics" and published in the mentioned document. (I.e. they where not invented by me!)

I was of the opinion, that the role of the work done here is to "translate" the documents published by other parts of the project (e.g. Semantics Team) to a technical description (= Json schema). 

And in the document there are ca. 12 value sets with > 50 values in total. Some of them will change frequently. So is it better to refer to the document or to include enums in the schema and change the schema frequently? Or to just include values, which are not supposed to change (e.g. male/female/unknown/.../other?)
Wo wants to decide this? Surely not me.